Server/docker-compose.swarm.yml

```diff
@@ -220,6 +220,28 @@ services:
       - 8089:8089
     expose:
       - 8089
+    volumes:
+      - ./:/usr/tenant
+      - /usr/tenant/node_modules
+    environment:
+      CITRINEOS_UTIL_DIRECTUS_USERNAME: 'admin@citrineos.com'
```

Comment:
i saw in another PR the username/password was going to be replaced with a CITRINEOS_UTIL_DIRECTUS_TOKEN -- if i should also update here then i can do so, let me know!

Reply:
ah, makes sense! for consistency i've updated to also use the token in the tenant module docker config, now that those changes are merged in main 🙌